src/ActionsMenuItem.test.js

```diff
@@ -112,15 +112,15 @@ it("Allows reenabling 'used' items that are hidden.", () => {
 });
 
 test.each([
-    // allowed, used , expectedDisabled
-    [  false  , false, false  ],
-    [  false  , true , false  ],
-    [  true   , false, false  ],
-    [  true   , true , true   ],
-])("The menu item is disabled when 'allowed' and 'used'.",
-    (allowed: boolean, used: boolean, expectedDisabled: boolean) => {
+    // disallowed, used , expectedDisabled
+    [  true , false, false  ],
+    [  true , true , false  ],
+    [  false, false, false  ],
+    [  false, true , true   ],
+])("The menu item is disabled when 'disallowed' or 'used'.",
```

Comment:
This description is not accurate. It was accurate as it was: "The menu item is disabled when 'allowed' and 'used'."